main_app/mail.py

```diff
@@ -14,16 +14,16 @@ def send_email(name, dest, link):
     server.starttls()
 
     # Enter your Email and Password
-    server.login("Email ID", "Password")
+    server.login("achyut662cs@gmail.com", "662cs@12345")
```

Comment:
Remove your Email ID and Password, and change it "Email ID" and "Password" respectively.